src/Components/Slider/Slider.js

```diff
@@ -9,9 +9,9 @@ const getWidth = () => window.innerWidth;
 /**
  * @function Slider
  */
-const Slider = (props) => {
+const Slider = ({width = getWidth(), items}) => {
```

Comment:
The `getWidth` is merely an alias of `window.innerWidth`, I'd suggest you to use the `window.innerWidth` constant rather than calling your function over and over. This way you will have a more explicit name for where the width is comming from.